team/quiet.rst

```diff
@@ -0,0 +1,40 @@
+
+############
+Quiet Friday
+############
+
+Starting on 2020-09-25 DMLT will commence **Quiet Friday**. 
+
+Guidelines
+==========
+The driver for the quiet day is to allow staff a block of time without interruptions or the need to be online and receptive. 
+By reducing traffic we will reduce FOMO on discussion topics. 
```

Comment:
Somebody's going to get confused about what “FOMO” means. Suggest rephrasing.

Reply:
Or add it to the list of acronyms. It is very commonly used. (although I agree, someone will get confused)